frontend/src/components/UI/Form/ComboBox/ComboBox.jsx

```diff
@@ -50,7 +50,7 @@ export const ComboBox = ({
 
         placeholder: (provided) => ({
             ...provided,
-            color: "#1b1b1b",
+            color: "#565C65",
```

Comment:
wonder if we have these colors in our `styles.scss`

Reply:
We might but seemingly the react-select api only exposes the styles to override